index.d.ts

```diff
@@ -0,0 +1,50 @@
+export as namespace EventEmitter;
+
+type ListenerFn = (...args: Array<any>) => void;
+
+/**
+  * Minimal `EventEmitter` interface that is molded against the Node.js
+  * `EventEmitter` interface.
+  */
+export class EventEmitter {
+  prefix: string;
+
+  /**
+    * Return an array listing the events for which the emitter has registered
+    * listeners.
+    */
+  eventNames(): Array<string | symbol>;
+
+  /**
+    * Return the listeners registered for a given event.
+    */
+  listeners(event: string | symbol): Array<ListenerFn>;
+  listeners(event: string | symbol, exists: boolean): Array<ListenerFn> | boolean;
```

Comment:
In TypeScript definitions, you can provide overloaded typings for the same method.

One implementation only takes `event` and always returns an array of listeners. The overload takes both parameters and either returns an array of listeners or a boolean, depending on the value of `exists`.

It is a best practice to put the most specific overload first, so I did swap the order of these in  365bef6.